Update evaluate module (no ic) (#50)
WayScience/phenotypic_profiling_model: 3.evaluate_model/README.md

In this module, we evaluate the final and shuffled baseline ML models.

After training the models in [2.train_model](../2.train_model/), we use these models to predict the labels of the training and testing datasets and evaluate their predictive performance.

> Evaluations are done on each model for each combination of model type (final, shuffled baseline), feature type (CP, DP, CP_and_DP), balance type (balanced, unbalanced), dataset type (ic, no_ic) and dataset (train, test).

Reviewer: ```suggestion
We evaluate each model for each combination of model type (final, shuffled baseline), feature type (CP, DP, CP_and_DP), balance type (balanced, unbalanced), dataset type (ic, no_ic) and dataset (train, test).
```

Active voice :)
